components/Profile/Page.js

```diff
@@ -565,6 +582,13 @@ const LoadedProfile = props => {
                   errors={errors}
                   dirty={dirty}
                 />
+                <div>
+                  <SubscribeMenu
+                    label='Profil Abonnieren'
+                    showAuthorFilter={true}
+                    subscriptions={{ nodes: Array.of(user.subscribedByMe) }}
```

Comment:
```suggestion
                    subscriptions={{ nodes: [user.subscribedByMe] }}
```
(I did end up checking and we do have the IE11 polyfill via next.js but I prefer using syntax instead of methods when possible)